### DISCUSSION TITLE
PDFブックマークに「タイトルページ」を入力する方法は?

### DISCUSSION THREAD

**Discussion by jasondee1992** [kmuto/review#1862] [Q&A] [answered]
どうも、

タイトル ページを PDF ブックマークに追加する方法を教えてください。 「rake pdf」を実行すると、タイトル ページの出力がタイトル ページに表示されません。

前もって感謝します。

![image](https://user-images.githubusercontent.com/49851974/190391152-3a0c578d-f434-45d5-ba8e-6f864b823853.png)

**Comment by kmuto** [comment, marked as answer]
TeX PDFに自動で入るもの以外のしおり(ブックマーク)を追加するには、`\pdfbookmark[level]{caption}{uniqueID}` を該当ページに入れることになります(see `texdoc hyperref`)。
Re:VIEWのtitlepageの場合、`\reviewtitlepagecont` のimplementationのところに入れるのがよさそうに思います。`\reviewtitlepagecont` は以前に review-custom.sty でカスタムしたものを使っていましたよね。

```
 ...
    \def\reviewtitlepagecont{%
       \begin{titlepage}
       \thispagestyle{empty}
       \pdfbookmark[1]{TITLE}{titlepage}
 ...
```

**Reply by jasondee1992** [reply]
ありがとうございます
